### ISSUE TITLE
[BUG]: Crashing is not defined in several test cases

### ISSUE THREAD

**Issue by CsatariGergely** [cncf/cnf-testsuite#1280]
**Describe the bug**
Crashing is a criteria in several test cases, but it is not defined what is a crash. 
Test cases are: 
- [Test](https://github.com/cncf/cnf-testsuite/blob/main/USAGE.md#heavy_check_mark-test-if-the-cnf-crashes-when-network-latency-occurs) if the CNF crashes when network latency occurs
- [Test](https://github.com/cncf/cnf-testsuite/blob/main/USAGE.md#heavy_check_mark-test-if-the-cnf-crashes-when-disk-fill-occurs) if the CNF crashes when disk fill occurs
- [Test](https://github.com/cncf/cnf-testsuite/blob/main/USAGE.md#heavy_check_mark-test-if-the-cnf-crashes-when-pod-delete-occurs) if the CNF crashes when pod delete occurs
- [Test](https://github.com/cncf/cnf-testsuite/blob/main/USAGE.md#heavy_check_mark-test-if-the-cnf-crashes-when-pod-memory-hog-occurs) if the CNF crashes when pod memory hog occurs
- [Test](https://github.com/cncf/cnf-testsuite/blob/main/USAGE.md#heavy_check_mark-test-if-the-cnf-crashes-when-pod-io-stress-occurs) if the CNF crashes when pod io stress occurs
- [Test](https://github.com/cncf/cnf-testsuite/blob/main/USAGE.md#heavy_check_mark-test-if-the-cnf-crashes-when-pod-network-corruption-occurs) if the CNF crashes when pod network corruption occurs
- [Test](https://github.com/cncf/cnf-testsuite/blob/main/USAGE.md#heavy_check_mark-test-if-the-cnf-crashes-when-pod-network-duplication-occurs) if the CNF crashes when pod network duplication occurs

**To Reproduce**

1. Go to https://github.com/cncf/cnf-testsuite/blob/main/USAGE.md
2. Read descriptions
3. See error

**Expected behavior**
A clear definition what crash is. 

**Screenshots**
N/A

**Device (please complete the following information):**

- All

**Comment by taylor** [comment]
Many of these testes are resiliency tests driven by the Litmus Chaos test experiments.  The usage guide has drop downs with provides some information.  There are links to the experiments which provide additional details on what is being tested.

At the moment the test suite is not checking that an application functions with e2e tests.  What is expected is that the readiness and liveness probes will be successful after a chaos experiment ends.

**Comment by CsatariGergely** [comment]
"What is expected is that the readiness and liveness probes will be successful after a chaos experiment ends." maybe this should be explained in the description.

**Comment by kosstennbl** [comment]
> "What is expected is that the readiness and liveness probes will be successful after a chaos experiment ends." maybe this should be explained in the description.

Agree, current documentation has only one usage of "crash" in that ambiguous meaning, but litmus chaos tests mostly use "should continue to function", which is not much clearer.
